src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -1155,8 +1155,10 @@ void grpc_chttp2_add_incoming_goaway(grpc_chttp2_transport* t,
       gpr_log(GPR_INFO, "transport %p got goaway with last stream id %d", t,
               last_stream_id));
   /* We want to log this irrespective of whether http tracing is enabled */
```

Comment:
This log line is contradictory to the change. Can you please fix it?

Reply:
ACK. Done!